GraphQLUpload.mjs

```diff
@@ -66,7 +66,7 @@ import Upload from "./Upload.mjs";
  * {@linkcode GraphQLUpload} scalar resolver value promise resolves:
  *
  * ```ts
- * import type { FileUpload } from "graphql-upload/processRequest.mjs";
+ * import { FileUpload } from 'graphql-upload/Upload.mjs';
```

Comment:
It's not correct to change the path for the `FileUpload` type; the typedef originates in `processRequest.mjs`:

https://github.com/jaydenseric/graphql-upload/blob/e01b5d5541760d529b06c900883c5fa7febcff00/processRequest.mjs#L352

As you can see, there is no `FileUpload` typedef exported in the module `Upload.mjs`:

https://github.com/jaydenseric/graphql-upload/blob/v16.0.2/Upload.mjs

Reply:
![image](https://user-images.githubusercontent.com/71709946/234250752-53f7dfb1-642d-44d2-af96-144e876941fe.png)
![image](https://user-images.githubusercontent.com/71709946/234250856-717b708b-9821-4afa-97fb-ca42ee6d41f7.png)
![image](https://user-images.githubusercontent.com/71709946/234250891-40590a91-19fd-4f7f-a280-0b9816bcff32.png)

@jaydenseric thanks for replaying :) I removed the type package but know for some reason I don't get the type from the type script as the pics above any idea why dose it not giving me the type?